bee-ternary/src/lib.rs

```diff
@@ -734,6 +764,57 @@ impl<T: RawEncodingBuf> TritBuf<T> {
     }
 }
 
+impl TritBuf<T3B1Buf> {
+    /// Pad the trit buffer with [`Btrit::Zero`] until the buffer's length is a multiple of 3.
+    ///
+    /// This method is often used in conjunction with [`Trites::as_trytes`].
+    pub fn pad_zeros(&mut self) {
+        while self.len() % 3 != 0 {
+            self.push(Btrit::Zero);
+        }
+    }
+
+    /// Pad the trit buffer with [`Btrit::Zero`] until the buffer's length is a multiple of 3.
+    ///
+    /// This method is often used in conjunction with [`Trites::as_trytes`].
+    pub fn padded_zeros(mut self) -> Self {
```

Comment:
Why do we need both ?

Reply:
I imagine this operation would be common immediately after creation so this allows it to be used inline